content/en/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -0,0 +1,117 @@
+In our example, it contains Istio CA's signing certificate, so `cert-chain.pem` is the same as `ca-cert.pem`.
+Note that if your `ca-cert.pem` is the same as `root-cert.pem`, the `cert-chain.pem` file should be empty.
+
+These files are ready to use in the `samples/certs/` directory.
+
+  {{< tip >}}
+  The default Istio's CA installation sets [command line options](/docs/reference/commands/istio_ca/index.html) to configure the location of certificates and keys based on the predefined secret and file names used in the command below (i.e., secret named `cacert`, root certificate in a file named `root-cert.pem`, Citadel key in `ca-key.pem`, etc.)
+  You must use these specific secret and file names, or reconfigure Istio's CA when you deploy it.
+  {{< /tip >}}
+
+The following steps plug in the certificates and key into a Kubernetes secret,
+which will be read by Istio's CA:
+
+1.  Create a secret `cacerts` including all the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`:
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    $ kubectl create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem \
+        --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem \
+        --from-file=samples/certs/cert-chain.pem
+    {{< /text >}}
+
+1.  Deploy Istio using the `demo` profile and with `global.mtls.enabled` set to `true`.
+
+    Istio's CA will read certificates and key from the secret-mount files.
+
+    {{< text bash >}}
+    $ istioctl manifest apply --set profile=demo --set values.global.mtls.enabled=true
+    {{< /text >}}
+
+## Verifying the certificates
+
+In this section, we verify that workload certificates are signed by the certificates that we plugged into the CA.
+This requires you have `openssl` installed on your machine.
+
+1. Deploy the `httpbin` and `sleep` sample services.
+
+    {{< text bash >}}
+    $ kubectl create ns foo
+    $ kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml) -n foo
+    $ kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml) -n foo
+    {{< /text >}}
+
+1.  Retrieve the certificate chain of `httpbin`.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c istio-proxy -n foo -- openssl s_client -showcerts -connect httpbin.foo:8000 > httpbin-proxy-cert.txt
+    {{< /text >}}
+
+    Open `httpbin-proxy-cert.txt`, which was created with the above command, and save the three certificates in it
+    to `proxy-cert-0.pem`, `proxy-cert-1.pem`, and `proxy-cert-2.pem`, respectively.
+    A certificate starts with `-----BEGIN CERTIFICATE-----` and ends with `-----END CERTIFICATE-----`.
+
+1.  Verify the root certificate is the same as the one specified by the administrator:
+
+    {{< text bash >}}
+    $ openssl x509 -in @samples/certs/root-cert.pem@ -text -noout > /tmp/root-cert.crt.txt
```

Comment:
What is "@" for?

Reply:
It will create a URL link to https://raw.githubusercontent.com/istio/istio/release-1.5/samples/certs/root-cert.pem.

Reply:
I see. Thanks.